frontend/src/concepts/topology/factories.ts

```diff
@@ -31,3 +33,21 @@ export const pipelineComponentFactory: ComponentFactory = (kind, type) => {
       return undefined;
   }
 };
+
+export const pipelineGroupsComponentFactory: ComponentFactory = (kind, type) => {
```

Comment:
Why do we need a new factory? We could just add the `EXECUTION_TASK_NODE_TYPE` to the `pipelineComponentFactory`, the rest is duplicated.

Reply:
Good point, thank you - updated